rules/xcodeproj.bzl

```diff
@@ -52,6 +64,8 @@ def _xcodeproj_aspect_impl(target, ctx):
     deps += getattr(ctx.rule.attr, "infoplists", [])
     deps.append(getattr(ctx.rule.attr, "entitlements", None))
```

Comment:
insted of checking for `None` all over the place, why don't we only `append` the entitlements target here conditionally?

```starlark
entitlements = getattr(ctx.rule.attr, "entitlements", None)
if entitlements:
    deps.append(entitlements)
```

Reply:
so if entitlements:  will fail if entitlements  is None?
I wonder if other places will add None to it upstream.